api/WebGL2RenderingContext.json

```diff
@@ -13,7 +13,7 @@
           },
           "edge": "mirror",
           "firefox": {
-            "version_added": "51"
+            "version_added": "80"
```

Comment:
So I think this collector suggestion is a major false claim that we need to fix. WebGL 2 was for sure in Firefox 51.

Given there is `reusableInstances.webGL2` which has a lot of dependencies, many collector suggestions are from this wrong Firefox 80 claim. I'm not sure where the bug is, though. Can you get WebGL running in Firefox 51 in manual testing?

Reply:
Okay, I figured out where the issue is.  In Firefox 51, I'm able to create a WebGL2 context no problem.  In Firefox 79, however, I am getting an error saying "Refused to create WebGL2 context because of blacklist entry: FEATURE_FAILURE_UNKNOWN_DEVICE_VENDOR".  (These are both run in BrowserStack.)  So, it seems like there's some hardware or hardware polling issue going on in a range of Firefox versions on BrowserStack.

I've added overrides to mitigate this issue.